sdk/eventhub/event-processor-host/changelog.md

```diff
@@ -1,89 +1,110 @@
+### 2.1.1
+
+- Updates minimum version of `@azure/event-hubs` to version 2.1.4.
+  This update brings in improvements when attempting to reconnect after a transient failure.
+  See the [changelog](https://github.com/Azure/azure-sdk-for-js/blob/%40azure/event-hubs_2.1.4/sdk/eventhub/event-hubs/changelog.md#2019-12-11-214)
+  for `@azure/event-hubs` for a full list of the improvements.
```

Comment:
> See the [changelog for `@azure/event-hubs`](https://github.com/Azure/azure-sdk-for-js/blob/%40azure/event-hubs_2.1.4/sdk/eventhub/event-hubs/changelog.md#2019-12-11-214)
for a full list of the improvements in the updates since version 2.1.1